connman/vpn/plugins/openvpn.c

```diff
@@ -357,38 +366,52 @@ static int ov_save(struct vpn_provider *provider, GKeyFile *keyfile)
 static int task_append_config_data(struct vpn_provider *provider,
 					struct connman_task *task)
 {
-	const char *option;
+	const char *option = NULL;
```

Comment:
This should be moved inside the loop; now string values can leak to bool side.

Reply:
Yep, forgot that completely when changing the arg adding part. Thanks.

Reply:
My thinking was the move the whole variable there, not used outside the loop block; this works too though.

Reply:
Well, now you mentioned it, that looks even better. Updated the review fix commit.